_positions/epa-fellow-tapd-technical-architect.md

```diff
@@ -0,0 +1,52 @@
+
+In addition to having a technical skills and experience in modern technical architecture and cloud computing platforms an ideal candidate must also have a broad understanding of open source software and standards to promote cost-effective adoption of commodity services as well as promote interoperability and automation within and across the full technology stack (applications and infrastructure).
+
+You must also understand enterprise and industry (DevOps) best practices for managing and streamlining change management processes. You will be a critical component of the team responsible for delivering strategic services and consulting to our partners within the EPA. You will work alongside executive leadership as well as our users to provide technical thought leadership to deliver modern technical and data infrastructure.
+
+##Key Objectives:
+**You will serve as the innovative technical leader in the development of agency modernization strategies for the the highly complex and significant systems across the EPA.**
+- Perform hands-on solution design, solution architectures, architecture roadmaps, prototyping, proof-of-concepts, and development tasks
+- Deliver strategies with an API first mindset using reusable components and operational strategies leveraging cloud services to support scalability, availability, performance, monitoring, backup, etc, to improve the security, resiliency, and agility of infrastructure and applications
+- Ensure strategic alignment of technical design/architecture to meet business growth and direction
+- Leverage native cloud capabilities such as data storage, performance, and security enhancements, as well as auto-scaling and performance/capacity management to promote technical improvements in the defendability and cost efficiency of the agency infrastructure services
+- Contribute high-quality, well-tested, maintainable infrastructure configurations, including code-based automation, to successfully impact the efficiency and effectiveness across the entire project lifecycle, using modern software development, open source, and industry best practices
+
+**You will promote the adoption of modern technology practices through teaching, coaching, and knowledge sharing.**
+- Practice and enthusiastically share agile and DevOps methodologies throughout all stages of the project lifecycle and infrastructure maintenance
+- Deliver educational workshops to accelerate learning and adoption of agile development and DevOps principles and practices
+- Inspire honesty and integrity through open communication and liberal knowledge sharing across a multi-disciplinary teams
```

Comment:
@rooneywp Same question as in the last PR: one team or multiple? There's a mismatch.